src/domains/global/sagas.ts

```diff
@@ -104,14 +105,23 @@ function* injectPosthog(machineGuid: string, personGuid?: string) {
       }
     },
   })
+  const shouldMaskReferrer = !isDemo && !isAllowedReferrer(document.referrer)
   window.posthog.register(
     // remove properties with unavailable values
     filter((value) => value !== undefined && value !== null,
       {
         $ip: "127.0.0.1",
-        $current_url: "agent dashboard",
-        $pathname: "netdata-dashboard",
-        $host: "dashboard.netdata.io",
+        $current_url: isDemo ? null : "agent dashboard",
+        $pathname: isDemo ? null : "netdata-dashboard",
+        $host: isDemo ? null : "dashboard.netdata.io",
+
+        $initial_referring_domain: shouldMaskReferrer ? "127.0.0.1" : null,
+        $initial_referrer: shouldMaskReferrer ? "http://127.0.0.1" : null,
+        $referring_domain: shouldMaskReferrer ? "127.0.0.1" : null,
+        $referrer: shouldMaskReferrer ? "http://127.0.0.1" : null,
```

Comment:
Can we use "masked" instead of "http://127.0.0.1"

Reply:
sure, done in all 4 lines that you mentioned